openreview/tools.py

```diff
@@ -1,29 +1,29 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
 import openreview
 import re
 import datetime
 
 super_user_id = 'OpenReview.net'
 
-def get_profile(client, value):
-    profile = None
-    try:
-        profile = client.get_profile(value)
-    except openreview.OpenReviewException as e:
-        # throw an error if it is something other than "not found"
-        if e[0][0] != 'Profile not found':
-            print "OpenReviewException: {0}".format(e)
-            return e
-    return profile
-
```

Comment:
Why remove this?  I've used it.

Reply:
it seems redundant with the client's get_profile function. The only difference seems to be that this function returns the exception, rather than just raising it, which I think is unconventional behavior.

Reply:
The difference is that it doesn't raise exceptions if the profile isn't found, it only raises exceptions if there is something wrong with the profile. It really should raise the exception if it's not the "not found" error.

Reply:
I agree with Pam, we should change the original get_profile function, for Not Found errors return None instead of an exception. 

It easier to handle the None values

Reply:
ok, I think we're all in agreement then. I just didn't think it belonged in the tools module.